kura/test/org.eclipse.kura.linux.position.test/src/test/java/org/eclipse/kura/linux/position/PositionServiceTest.java

```diff
@@ -184,7 +185,7 @@ public void testActivateDeactivate() throws IOException, InterruptedException {
 
         // it takes some time for the event to be sent...
         synchronized (o) {
-            o.wait(1000);
+            o.wait(10000);
```

Comment:
The worst this may do is prolong the test execution for 9 s. But if it really takes more than 1 s for the service to send event, it solves the problem.